avocado/utils/pci.py

```
@@ -376,6 +376,91 @@
+    Check whether the PCI device supports MSI-X and if it is currently enabled.
+
+    :param device: full pci address including domain (0000:03:00.0)
+    :return: True if supported, False otherwise
+    """
+    cmd = f"lspci -vvs {device} | grep -e 'MSI-X:'"
+    output = process.run(cmd, ignore_status=True, shell=True, sudo=True)
+    if output.stderr.decode("utf-8") != "":
+        raise ValueError(f"{output}")
+    if output.stdout_text.strip() == "":
+        return False
+    return True
```

Ruff (0.12.2)

Return the condition `output.stdout_text.strip() != ""` directly

Replace with `return output.stdout_text.strip() != ""`

(SIM103)